pkg/notes/notes_map.go

```diff
@@ -75,44 +75,45 @@ func ParseReleaseNotesMap(mapPath string) (*[]ReleaseNotesMap, error) {
 	return &notemaps, nil
 }
 
-// ReleaseNotesMap Represents
+// ReleaseNotesMap holds the changes that will be applied to the notes
+// the struct has double annotation because of this: https://play.golang.org/p/C7QM1varozy
 type ReleaseNotesMap struct {
 	// Pull request where the note was published
 	PR int `json:"pr"`
 	// SHA of the notes commit
-	Commit      string `json:"commit"`
+	Commit      string `json:"commit,omitempty" yaml:"commit,omitempty"`
 	ReleaseNote struct {
 		// Text is the actual content of the release note
-		Text *string `json:"text,omitempty"`
+		Text *string `json:"text,omitempty" yaml:"text,omitempty"`
 
 		// Docs is additional documentation for the release note
-		Documentation *[]*Documentation `json:"documentation,omitempty"`
+		Documentation *[]*Documentation `json:"documentation,omitempty" yaml:"documentation,omitempty"`
 
 		// Author is the GitHub username of the commit author
-		Author *string `json:"author,omitempty"`
+		Author *string `json:"author,omitempty" yaml:"author,omitempty"`
 
 		// Areas is a list of the labels beginning with area/
-		Areas *[]string `json:"areas,omitempty"`
+		Areas *[]string `json:"areas,omitempty" yaml:"areas,omitempty"`
```

Comment:
Are those `yaml` tags really required? I thought they would be inferred from the `json` tags already.

Reply:
I thought so too. But after cracking my head for a while I managed to isolate the problem. It seems that the YAML output renders differently when values are missing when annotated with yaml: and json: 

I linked my test case in the comment: https://play.golang.org/p/C7QM1varozy
I do not know if its a bug or a feature (or maybe I'm doing something wrong that escapes me?)

Reply:
Hm that's indeed different :thinking: